api.md

```diff
@@ -105,3 +105,18 @@ Add something to the end of an array.
 ```javascript
 dispatch(arrayAdd('app.todos', 'New to do');
 ```
+
+## objectMerge
+
+Add or edit an object by merging another into it.
```

Comment:
Please mention the deep merge algorithm and that arrays are copied shallow. The "add an object" seems confusing to me. Maybe stick to "merge".